### ISSUE TITLE
Add a Feeder - Process

### ISSUE THREAD

**Issue by wsciaroni** [uah-cpe-sd-2020/myportion_app#23]
## Background

In order for the feeder to be controlled from the app, it needs to get connected to the WiFi.  In order to do this, we want to have a setup process in the App.  We want the following steps:

1. Plug in feeder
2. Look in app for `+` add feeder button
3. Connect to the device via WiFi
4. Enter or select which WiFi you want the device to use for internet access
5. Have the app disconnect from the feeder and have the feeder connect to the WiFi
6. Have the Serial Number of the feeder associated with the user that set up the account.  If it's already there, then leave it as is.  If it is in anyone else's account, purge it.

## Acceptance Criteria

This story covers the scope of having the app walk the user through the whole setup process of adding a feeder.  Specifically, make the routes available in the app so that we can add the functionality in another story.

This story DOES NOT cover the scope of having the feeder do the whole connection.  However, these are highly intertwined and should likely be developed in parallel.

**Comment by wsciaroni** [comment]
@CollierA13j Please make a note of what's blocking :)

**Comment by CollierA13j** [comment]
This issue is blocked since it cannot be worked on until the we start working on the feeder itself

**Comment by wsciaroni** [comment]
@CollierA13j I am now working on setting up the hardware side of this issue.

I would think we need to have the #26 implemented so that someone can implement the first part of this story: 
- Getting the UI to the wifi selection page.

So this is blocked by #26

**Comment by CollierA13j** [comment]
Has this been completed?
